recipes/sdl/all/conanfile.py

```diff
@@ -379,6 +391,9 @@ def package_info(self):
 
         postfix = "d" if self.settings.os != "Android" and self.settings.build_type == "Debug" else ""
 
+        if self.version >= "2.0.24" and is_msvc(self) and not self.options.shared:
+            postfix += "-static"
```

Comment:
The name of the static debug library is `SDL2-staticd.lib`. The `d` is appended to `SDL2-static`.
So `NOT `SDL2d-static.lib`.

Reply:
done